src/Uno.UI/UI/Xaml/IFrameworkElement.cs

```diff
@@ -296,11 +296,14 @@ public static IFrameworkElement FindName(IFrameworkElement e, IEnumerable<View>
 		}
 
 		private static IFrameworkElement FindInFlyout(string name, Controls.Primitives.FlyoutBase flyoutBase)
-			=> flyoutBase switch
+		{
+			if (__LinkerHints.Is_Windows_UI_Xaml_Controls_MenuFlyout_Available && flyoutBase is MenuFlyout f)
 			{
-				MenuFlyout f => f.Items.Select(i => i.FindName(name) as IFrameworkElement).Trim().FirstOrDefault(),
-				Controls.Primitives.FlyoutBase fb => fb.GetPresenter()?.FindName(name) as IFrameworkElement
-			};
+				return f.Items.Select(i => i.FindName(name) as IFrameworkElement).Trim().FirstOrDefault();
```

Comment:
Does `Trim` filters nulls? In that case, I think the following should work and is shorter, and probably more readable?

```suggestion
				return f.Items.FirstOrDefault(i => i.FindName(name) is IFrameworkElement);
```

Reply:
Yes, trim excludes null, but it's not very useful here.